### DISCUSSION TITLE
21.8.2

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1468] [General]
## Releases Notes for 21.8.2
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v21.8.2/draw.io-21.8.2-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v21.8.2/draw.io-21.8.2-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v21.8.2/draw.io-universal-21.8.2.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v21.8.2/drawio-amd64-21.8.2.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v21.8.2/drawio-x86_64-21.8.2.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v21.8.2/drawio-x86_64-21.8.2.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Updates to [draw.io core 21.8.2](https://github.com/jgraph/drawio/blob/v21.8.2/ChangeLog). All changes from 21.7.6 to 21.8.2 are added in this build.
- Fixes #1411 

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v21.8.2'>21.8.2</a>.</em>

**Comment by AndreanRip** [comment]
Hello. After the update, the apps in my end stuck on the menu choose between
new diagram
open diagram
even after choose one, those menu still persist. Already tried re install a couple of them and it doesn't work. for now I use roll back and use the previous ver

**Reply by AndreanRip** [reply]
ah right, forgot to mention I use the windows installer. I tried the no need install and it still doesn't work

**Reply by b1506704** [reply]
Same thing happened to me. I have opened an issue by the way: https://github.com/jgraph/drawio-desktop/issues/1470

**Reply by MarkoMilos** [reply]
The same issue manifests on macOS (Ventura 13.5.1). 

Steps to reproduce:
1. Open draw.io
2. Select "Open existing diagram"
3. Find the diagram file that you want to open (saved locally on disk)
4. Open the diagram file

Expected result:
The diagram opens in the draw.io app and it is ready for editing.

Actual result:
The diagram opens in the background but it is overlayed with the dialog prompting the user to either "Open new diagram" or "Open existing diagram". The dialog cannot be closed by pressing the close button (x). Basically, this bug completely prevents the user from using draw.io offline (locally) since it is not possible to open&edit existing diagrams.
